fighthealthinsurance/middleware/AuditLoggingMiddleware.py

```
@@ -0,0 +1,167 @@
+
+    def process_response(
+        self, request: HttpRequest, response: HttpResponse
+    ) -> HttpResponse:
+        """Log the API access after response is generated."""
+        # Skip if path shouldn't be logged
+        if not _should_log_request(request.path):
+            return response
+
+        try:
+            self._log_api_access(request, response)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)